### DISCUSSION TITLE
To solve error

### DISCUSSION THREAD

**Discussion by Praveen-cyber-security** [community/community#138045] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

No such file or directory: 'Xvfb'

**Comment by khalil-sohail** [comment]
try in terminal:
                Xvfb -help
If the command is not found then you can usually install the xvfb package with the following command (or whichever package manager you generally use):

               sudo apt-get install xvfb
[xvfbwrapper](https://github.com/cgoldberg/xvfbwrapper) is Python wrapper for running a display inside X virtual framebuffer (Xvfb).

You can install it with:

               pip install xvfbwrapper

The error you are getting seems to be a result of either the command or module not being found.
